package.json

```diff
@@ -38,30 +38,25 @@
 		"promise"
 	],
 	"dependencies": {
-		"map-age-cleaner": "^0.1.3",
+		"map-age-cleaner": "^0.2.0",
 		"mimic-fn": "^4.0.0"
 	},
 	"devDependencies": {
-		"@ava/typescript": "^1.1.1",
-		"@sindresorhus/tsconfig": "^1.0.2",
-		"@types/serialize-javascript": "^4.0.0",
-		"ava": "^3.15.0",
-		"del-cli": "^3.0.1",
-		"delay": "^4.4.0",
-		"serialize-javascript": "^5.0.1",
-		"ts-node": "^10.1.0",
-		"tsd": "^0.13.1",
-		"typescript": "^4.3.5",
-		"xo": "^0.41.0"
+		"@sindresorhus/tsconfig": "^3.0.1",
```

Comment:
This changes the output:

```
file:///home/runner/work/mem/mem/node_modules/xo/node_modules/meow/index.js:85
			flag.default = flag.default ?? [];
			                             ^

SyntaxError: Unexpected token '?'
```

Is it time to drop Node 12 while we're at it? Last breaking version was almost 2 years ago https://github.com/sindresorhus/mem/releases/tag/v9.0.0

Reply:
Yes, we can target Node.js 16.